ministryofjustice/correspondence_tool_staff
# Update r102 for SAR IRs

## Description
Adds SAR IRs to the r102 report.

## Self-review checklist



### Screenshots


### Related JIRA tickets


### Deployment


### Manual testing instructions
- create some SAR IR / SAR ICO cases.
- click reports > create report > select SAR report > Appeals performance report > add dates > click create custom report.
- Download the report
- Click the down load link when available
- Check the report looks correct.
